service/generators.gradle

```diff
@@ -0,0 +1,45 @@
+dependencies {
+    implementation 'io.swagger.core.v3:swagger-annotations'
+    runtimeOnly 'org.webjars.npm:swagger-ui-dist:4.9.0'
+    swaggerCodegen 'io.swagger.codegen.v3:swagger-codegen-cli'
+
+    // Versioned by Spring:
+    implementation 'javax.validation:validation-api'
+    implementation 'org.webjars:webjars-locator-core'
+
+    annotationProcessor 'org.springframework.boot:spring-boot-configuration-processor'
+}
+
+def artifactGroup = "${group}.axonserver"
+
+generateSwaggerCode {
+    inputFile = file('../service/src/main/resources/api/openapi.yml')
+    language = 'spring'
+    components = ['models', 'apis']
+    jvmArgs = ['--add-opens=java.base/java.util=ALL-UNNAMED']
+    additionalProperties = [
+        modelPackage     : "${artifactGroup}.model",
+        apiPackage       : "${artifactGroup}.api",
+        dateLibrary      : 'java11',
+        java8            : true,
+        interfaceOnly    : 'true',
+        useTags          : 'true',
+        errorOnUnknownEnums : 'true',
+        springBootVersion: dependencyManagement.managedVersions['org.springframework.boot:spring-boot']
```

Comment:
I recommend using `modelNamePrefix  : 'Api',`
That will prefix all of the OpenApi generated model class with "Api". In code, it is good to know when you are using an API class vs an internal implementation class.